erica/erica_legacy/request_processing/erica_input/v1/erica_input.py

```diff
@@ -220,12 +220,12 @@ class EstData(BaseModel):
 
 class UnlockCodeRequestData(BaseModel):
     idnr: str
-    dob: date
+    date_of_birth: date = Field(alias='dob')
 
 
 class UnlockCodeActivationData(BaseModel):
     idnr: str
-    unlock_code: str
+    freischalt_code: str = Field(alias='unlock_code')
     elster_request_id: str
 
```

Comment:
That's why we do not have to adapt the webapp request. 

We set the alias and this will therefore make the translation for us. You can see that I did not change test_integration.py, which tests v1 of the API.